src/components/Radio/stories.js

```diff
@@ -31,16 +116,21 @@ storiesOf("Radio", module).add("Radio disabled", () => (
 
 storiesOf("Radio", module).add("Radio preselected", () => (
   <div>
-    <Radio name="group1" defaultChecked>
+    <Radio name="group1" readOnly checked>
```

Comment:
why readOnly? Would checked  work on its own? Maybe create a separate story for readOnly?

Reply:
yeah... this was one of the errors/warnings it displayed said it should be either readOnly when i was trying to get the reset working, so this was just trying to please that error